tests/test_image_structure.py

```diff
@@ -0,0 +1,74 @@
+import rasterio
+
+from rasterio.enums import Compression, Interleaving
+
+
+def test_enum_compression_JPEG():
```

Comment:
@brendan-ward these tests of the enums are mainly to guard against future regressions. They look clunky because I didn't presume all lower or all upper case at the beginning. It did turn out that way, but I decided to leave the tests as they are.